### ISSUE TITLE
Getting warning while installing via yarn for react-is and stylis, even though they are there in node modules

### ISSUE THREAD

**Issue by rahul3103** [styled-components/styled-components#2874]
# System:
 - OS: macOS Mojave 10.14.5
 - CPU: (4) x64 Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz
 - Memory: 50.37 MB / 8.00 GB
 - Shell: 5.3 - /bin/zsh
## Binaries:
 - Node: 12.13.0 - ~/.nvm/versions/node/v12.13.0/bin/node
 - Yarn: 1.19.1 - /usr/local/bin/yarn
 - npm: 6.12.0 - ~/.nvm/versions/node/v12.13.0/bin/npm
## npmPackages:
 - babel-plugin-styled-components: ^1.10.6 => 1.10.6
 - styled-components: ^5.0.0-rc.1 => 5.0.0-rc.1

Why I am getting this warning, even though I can see packages in node_modules

```
yarn add styled-components@beta
yarn add v1.19.1
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "styled-components > stylis-rule-sheet@0.0.10" has unmet peer dependency "stylis@^3.5.0".
warning " > styled-components@5.0.0-rc.1" has unmet peer dependency "react-is@>= 16.8.0".
[4/4] 🔨  Building fresh packages...
success Saved 1 new dependency.
info Direct dependencies
└─ styled-components@5.0.0-rc.1
info All dependencies
└─ styled-components@5.0.0-rc.1
✨  Done in 3.19s.
```

**Comment by probablyup** [comment]
Will be fixed in v5 final or if there is a next RC

**Comment by WayneEllery** [comment]
I'm still getting the `react-is`: `warning " > styled-components@5.0.0" has unmet peer dependency "react-is@>= 16.8.0"`

**Comment by kitten** [comment]
@WayneEllery react-is must unfortunately remain a peer dependency, which means you’ll have to install it separately. But don’t worry, yarn will deduplicate it.

This is how peer dependencies work, the presence of a package is not enough to fulfil a peer dependency. Instead you have to explicitly take on the dependency yourself

**Comment by janKir** [comment]
> react-is must unfortunately remain a peer dependency, which means you’ll have to install it separately.
I think this should be mentioned in the "Installation" docs.

**Comment by probablyup** [comment]
It is

**Comment by janKir** [comment]
> It is

Yes, in the README. But I cannot find it on the [Website](https://styled-components.com/docs/basics#installation).

**Comment by probablyup** [comment]
Ah ok, that particular page needs to be updated. It was a new requirement
for v5:
https://styled-components.com/docs/faqs
